sdk/python/feast/registry_server.py

```diff
@@ -336,6 +337,47 @@ def GetInfra(self, request: RegistryServer_pb2.GetInfraRequest, context):
             project=request.project, allow_cache=request.allow_cache
         ).to_proto()
 
+    def ApplyPermission(
+        self, request: RegistryServer_pb2.ApplyPermissionRequest, context
+    ):
+        self.proxied_registry.apply_permission(
+            permission=Permission.from_proto(request.permission),
+            project=request.project,
+            commit=request.commit,
+        )
+        return Empty()
+
+    def GetPermission(self, request: RegistryServer_pb2.GetPermissionRequest, context):
+        permission = self.proxied_registry.get_permission(
+            name=request.name, project=request.project, allow_cache=request.allow_cache
+        ).to_proto()
+
+        permission.project = request.project
+        return permission
+
+    def ListPermissions(
+        self, request: RegistryServer_pb2.ListPermissionsRequest, context
+    ):
+        permissions_proto = []
+        permissions = self.proxied_registry.list_permissions(
+            project=request.project,
+            allow_cache=request.allow_cache,
+        )
+        for p in permissions:
+            permission_proto = p.to_proto()
+            permission_proto.project = request.project
+            permissions_proto.append(permission_proto)
+
+        return RegistryServer_pb2.ListPermissionsResponse(permissions=permissions_proto)
```

Comment:
Why don't we use the regular pattern:
```py
        return RegistryServer_pb2.ListPermissionsResponse(
            permissions=[
                permission.to_proto()
                for permission in self.proxied_registry.list_permissions(
                    project=request.project,
                    allow_cache=request.allow_cache,
                )
            ]
        )
```

BTW: do we really need to set the `project` field? I can't see that this is managed in the equivalent methods for other resource types

Reply:
You are right, I just didn't want to take any chances, will remove it